### ISSUE TITLE
Feature Request: Disable Cookies, Whitelist with Extension

### ISSUE THREAD

**Issue by monkeymagic29** [Cookie-AutoDelete/Cookie-AutoDelete#126]
### Describe the Bug/Feature

Hi. I had been using AutoDestruct for a while on my Androids. You see, I'm this minimum-access, blacklist-all-first-whitelist-one-by-one-later. Which is my policy in setting up NoScript and UBO. 
Autodestruct have been functioning more as a GUI substitute for a feature easily found on Firefox desktop client but not Android client: temporarily/permanently whitelist cookies from select sites while setting client on disable cookies. The autodestruct is just cherry on top for me.
So, may I ask you to try to maintain similar paradigm? Now it seems like I have to allow every unwanted cookie to get in first rather than let them sit it out outside.

In short: I want to disable cookies on Firefox Android, and C-AD can later whitelist (permanently and/or temporarily) those I select to join my browsing party as Autodestruct used to do in my devices. Cheers!!

**Comment by mrdokenny** [comment]
Duplicate of #95
